include/seqan3/alignment/pairwise/detail/policy_optimum_tracker.hpp

```diff
@@ -96,8 +158,12 @@ class policy_optimum_tracker
      * algorithm requires it, for example when a local alignment shall be computed.
      */
     template <typename cell_t>
-    decltype(auto) track_cell(cell_t && cell, matrix_coordinate SEQAN3_DOXYGEN_ONLY(coordinate)) noexcept
+    decltype(auto) track_cell(cell_t && cell, matrix_coordinate_type coordinate) noexcept
     {
+        if (test_every_cell)
+            compare_and_set_optimum(std::forward_as_tuple(optimal_score, optimal_coordinate),
+                                    std::forward_as_tuple(cell.optimal_score(), std::move(coordinate)));
```

Comment:
This expression is written 3 times, is there no way to dry this?

Reply:
Sure, you can always add one more function to it. But is that really a problem here?

Reply:
The reason was, that I basically want to tell that score and coordinate are associated and that the resulting operation is a binary operation which is more natural than passing 4 arguments.

Reply:
I kind of understand the "more" natural thing, because you logically group them.

But I think it obfuscates which variables are lvalue and which ones are not and makes it in general harder to understand the semantics of the function. 

Since we bundled it into one call (`invoke_comparator`), it is not really harder to understand the semantics any more. On the other hand this also takes the point of needing it to be more natural, since we only have a single invocation, and we don't need a "natural" feeling for a bypassing piece of code.

Reply:
This is still open, I personally would prefer the explicit argument list, but let's sneakingly wait if the 2nd reviewer writes "WTH?!"

Reply:
ah ok. I didn't see this. No I can change this. No hard feelings to call it with four parameters.